examples/vision/ipynb/randaugment.ipynb

```diff
@@ -0,0 +1,590 @@
+    "to do any separate preprocessing on our dataset and is specifically very useful for\n",
+    "deployment purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "def get_training_model():\n",
+    "    resnet50_v2 = tf.keras.applications.ResNet50V2(\n",
+    "        weights=None,\n",
+    "        include_top=True,\n",
+    "        input_shape=(IMAGE_SIZE, IMAGE_SIZE, 3),\n",
+    "        classes=10,\n",
+    "    )\n",
+    "    model = tf.keras.Sequential(\n",
+    "        [\n",
+    "            layers.Input((IMAGE_SIZE, IMAGE_SIZE, 3)),\n",
+    "            layers.experimental.preprocessing.Rescaling(scale=1.0 / 127.5, offset=-1),\n",
+    "            resnet50_v2,\n",
+    "        ]\n",
+    "    )\n",
+    "    return model\n",
+    "\n",
+    "\n",
+    "print(get_training_model().summary())\n",
+    ""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "We will train this network on two different versions of our dataset:\n",
+    "\n",
+    "* One augmented with RandAugment.\n",
+    "* Another one augmented with `simple_aug`.\n",
+    "\n",
+    "Since RandAugment is known to enhance the robustness of models to common perturbations\n",
+    "and corruptions, we will also evaluate our models on the CIFAR-10-C dataset, proposed in\n",
+    "[Benchmarking Neural Network Robustness to Common Corruptions and Perturbations](https://arxiv.org/abs/1903.12261)\n",
+    "by Hendrycks et al. The CIFAR-10-C dataset\n",
+    "consists of 19 different image corruptions and perturbations (for example speckle noise,\n",
+    "fog, Gaussian blur, etc.) that too at varying severity levels. For this example we will\n",
+    "be using the following configuration:\n",
+    "[`cifar10_corrupted/saturate_5`](https://www.tensorflow.org/datasets/catalog/cifar10_corrupted#cifar10_corruptedsaturate_5).\n",
+    "The images from this configuration look like so:\n",
+    "\n",
+    "![](https://storage.googleapis.com/tfds-data/visualization/fig/cifar10_corrupted-saturate_5-1.0.0.png)\n",
+    "\n",
+    "In the interest of reproducibility, we serialize the initial random weights of our shallow\n",
+    "network."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "initial_model = get_training_model()\n",
+    "initial_model.save_weights(\"initial_weights.h5\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "#1. Train model with RandAugment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "rand_aug_model = get_training_model()\n",
```

Comment:
models are actually the same, definitely trained on a dataset with different augmentations.

could be naming the model `model_with_rand_aug` would make it a bit more transparent for people?

Reply:
I make the distinction by naming them like `rand_aug_model` and `simple_aug_model`. Maybe I am missing out on something?